src/bbs2gh/Program.cs

```diff
@@ -117,11 +117,11 @@ private static async Task LatestVersionCheck(ServiceProvider sp)
 
             if (await versionChecker.IsLatest())
             {
-                Logger.LogInformation($"You are running the latest version of the bbs2gh extension [v{await versionChecker.GetLatestVersion()}]");
+                Logger.LogInformation($"You are running an up-to-date version of the bbs2gh extension [v{versionChecker.GetCurrentVersion().ToString(3)}]");
```

Comment:
When showing the version to the customer, we only want to show `major.minor.patch`, omitting the revision.